ui/pages/confirmations/components/confirm/info/shield-subscription-approve/subscription-details.tsx

```diff
@@ -0,0 +1,70 @@
+import {
+  Box,
+  BoxAlignItems,
+  BoxFlexDirection,
+  BoxJustifyContent,
+} from '@metamask/design-system-react';
+import React from 'react';
+import { ConfirmInfoSection } from '../../../../../../components/app/confirm/info/row/section';
+import { Text } from '../../../../../../components/component-library';
+import {
+  TextColor,
+  TextVariant,
+} from '../../../../../../helpers/constants/design-system';
+import { useI18nContext } from '../../../../../../hooks/useI18nContext';
+
+export const SubscriptionDetails = ({
+  approvalAmount,
+  showTrial,
+}: {
+  approvalAmount: string;
+  showTrial: boolean;
+}) => {
+  const t = useI18nContext();
+
+  const isMonthlySubscription = approvalAmount === '96';
```

Comment:
we will use data from subscription controller state in a subsequent pr here @tuna1207